data/json/obsoletion/obsolete-0.G-recipes.json

```diff
@@ -427,5 +427,15 @@
     "result": "xs_k_gambeson_hood",
     "type": "recipe",
     "obsolete": true
+  },
+  {
+    "type": "uncraft",
```

Comment:
Non-existing recipes just get deleted silently; you should be able to remove these entirely, (and these 0.G files shouldn't be updated for new obsoletions)